scripts/dev-start-legacy.sh

```diff
@@ -12,7 +12,7 @@ echo "Starting @ $(date)" > $PATH_ROOT/logs/dev.txt
 
 # run flask
 cd src/api/
-flask run --reload >> $PATH_ROOT/logs/api.txt 2>&1 & # run in bg and port logs
+DB_USER=marchdan DB_PASS=" "  flask run --reload >> $PATH_ROOT/logs/api.txt 2>&1 & # run in bg and port logs
```

Comment:
I'm guessing that DB_USER shouldn't probably be your own account?

Reply:
Crap, I thought I unstated that file XD Will fix on next push

Reply:
quick fix here, can you remove your specific ENV vars